jellyfin/JavascriptSubtitlesOctopus
# Simple Web Example

A simple web example showing how to instantiate the sub renderer. While looking to update the version in a project I'm working on from v2.1.1 of the `libass-wasm` npm package from the original repo. I wanted to take advantage of the great work that you are doing here on this repo, but seem that the project is no longer being merged back to the original repo. So thought I would give it a shot on your branch. However it looks like it's not working as it did previously. So I created this simple web example to make sure that there wasn't something fishy with my project. However it's currently not working, and throwing a worker error:

![image](https://user-images.githubusercontent.com/6342442/84007520-85f2b900-a9ab-11ea-877e-08e0eb6274e9.png)

Is there possibly something that has changed since v2.1.1, that no longer allows this to render on web? Tested on desktop Chrome 83.0.4103.61

Test Instructions
- In root folder start a simple server for the static assets "python -m SimpleHTTPServer 9003"
- Navigate to "http://localhost:9003/example/"
- Video should start playing, but no subtitles are visible
- Check debugger console and observe error
